libs/commons/List_.ml

```diff
@@ -17,6 +17,8 @@
 (* Faster and stack-safe List.map *)
 (*****************************************************************************)
 
+open Eq
```

Comment:
to be consistent, maybe we can do open Eq.Operators

so adding an extra Operators submodule in Eq,
and them in Common we can do include Eq.Operators.

Reply:
Sure